src/components/Counter.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './Counter.scss';
+
+export class Count extends React.Component {
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 1,
+    }));
+  }
+
+  add100 = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 100,
+    }));
+  }
+
+  increase = () => {
+    if (this.state.count % 5 !== 0) {
+      this.addOne();
+    } else {
+      this.addOne();
+      this.add100();
+    }
+  }
```

Comment:
`else ` тут лишний, можно сделать проще
```suggestion
  increase = () => {
    if (this.state.count % 5 === 0) {
      this.addO100();
    } 
      this.addOne();
    }
```